src/main/resources/db/changelog/changes/03-add-not-null-constraint-to-title.yaml

```diff
@@ -0,0 +1,9 @@
+databaseChangeLog:
+  - changeSet:
+      id: add-not-null-constraint-to-title
+      author: savchenko
+      changes:
+        - addNotNullConstraint:
```

Comment:
let's add `constraints: nullable: false` for `title` to existing changelog 01-create-book-table.yaml